apps/ejabberd/test/mongoose_rdbms_SUITE.erl

```diff
@@ -48,13 +50,25 @@ init_per_group(Group, Config) ->
 end_per_group(_, Config) ->
     Config.
 
+init_per_testcase(does_backoff_increase_to_a_point, Config) ->
+    DbType = ?config(db_type, Config),
+    meck_config(DbType, 2, 10),
+    meck_db(DbType),
+    meck_connection_error(DbType),
+    Config;
 init_per_testcase(_, Config) ->
     DbType = ?config(db_type, Config),
-    meck_config(DbType, ?KEEPALIVE_INTERVAL),
+    meck_config(DbType, ?KEEPALIVE_INTERVAL, ?MAX_INTERVAL),
     meck_db(DbType),
     Config.
 
-end_per_testcase(_, Config) ->
+end_per_testcase(does_backoff_increase_to_a_point, Config) ->
+    Db = ?config(db_type, Config),
+    meck_connection_error_unload(Db),
+    meck_unload(Db),
```

Comment:
Please either split it into `meck_db_unload` and `meck_config_unload` or at least rename it into `meck_config_and_db_unload`. :)